### PR TITLE
[DF-180] SAML auth moved to frontend

### PR DESCRIPTION
**Pull request by wwebec**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -47,11 +47,11 @@ before_deploy:
 deploy:
 - provider: script
   skip_cleanup: true
-  script: >-
-    ./gradlew bootWar dependencyCheckAggregate pushDockerImage publish -Pprod -Pzipkin -x test -Pversion=${TRAVIS_JOB_NUMBER}-SNAPSHOT -PaddLatestTagToDockerImage=true --stacktrace
-    && .ci-travis/push-version-bump.sh ${TRAVIS_JOB_NUMBER}-SNAPSHOT ${GIT_USER} ${GIT_PWD}
+  script: "./gradlew bootWar dependencyCheckAggregate pushDockerImage publish -Pprod -Pzipkin -x test -Pversion=`echo ${TRAVIS_BRANCH} | cut -c9-`-${TRAVIS_JOB_NUMBER}-SNAPSHOT"
+#    && .ci-travis/push-version-bump.sh ${TRAVIS_JOB_NUMBER}-SNAPSHOT ${GIT_USER} ${GIT_PWD}
```
Comment (pavax) [inline .travis.yml]: revert

File: src/main/java/ch/admin/seco/jobroom/config/LoginRedirectURISessionAttribute.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package ch.admin.seco.jobroom.config;
+
+import org.springframework.beans.factory.annotation.Value;
+
+public class LoginRedirectURISessionAttribute {
+
+    @Value("${jhipster.mail.base-url}")
```
Comment (pavax) [inline src/main/java/ch/admin/seco/jobroom/config/LoginRedirectURISessionAttribute.java]: change namespace: security.user.redirect-base-url + add it to the application.yml and the central config project
in central-config you could define
``` 
security.user.redirect-base-url = ${jhipster.mail.base-url}
```

Comment (wwebec) [reply src/main/java/ch/admin/seco/jobroom/config/LoginRedirectURISessionAttribute.java]: Added a property of [security user redirect base URL](https://github.com/alv-ch/central-config/pull/62/files)

File: src/main/java/ch/admin/seco/jobroom/config/LoginRedirectURISessionAttribute.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package ch.admin.seco.jobroom.config;
+
+import org.springframework.beans.factory.annotation.Value;
+
+public class LoginRedirectURISessionAttribute {
+
+    @Value("${security.user.redirect-base-url}")
```
Comment (pavax) [inline src/main/java/ch/admin/seco/jobroom/config/LoginRedirectURISessionAttribute.java]: can we still start the application locally even though this property is not set locally?

**Pull request merged**